v2.0/alter-sequence.md

```diff
@@ -4,24 +4,25 @@ summary:
 toc: false
 ---
 
-<span class="version-tag">New in v2.0:</span> The `ALTER SEQUENCE` [statement](sql-statements.html)...
+<span class="version-tag">New in v2.0:</span> The `ALTER SEQUENCE` [statement](sql-statements.html) applies a schema change to a sequence.
```

Comment:
This sounds awkward. Can we say "The `ALTER SEQUENCE` changes the name and increment values of a sequence"? The "applies a schema change" wording sounds right for a table or database, not sure it is applicable to sequences.

Reply:
edited